vulnerabilities/importers/redhat.py

```diff
@@ -75,21 +78,53 @@ def to_advisory(advisory_data):
     references = []
     if advisory_data.get("bugzilla"):
         bugzilla = advisory_data.get("bugzilla")
+        url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
+        bugzilla_data = requests.get(f"https://bugzilla.redhat.com/rest/bug/{bugzilla}").json()
+        bugzilla_severity_val = bugzilla_data["bugs"][0]["severity"]
+        bugzilla_severity = VulnerabilitySeverity(
+            system=scoring_systems["rhbs"],
+            value=bugzilla_severity_val,
+        )
+
         references.append(
             Reference(
-                url="https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla),
+                severities=[bugzilla_severity],
+                url=url,
                 reference_id=bugzilla,
             )
         )
 
-    for rhsa in advisory_data["advisories"]:
-        references.append(
-            Reference(
-                url="https://access.redhat.com/errata/{}".format(rhsa), reference_id=rhsa,
+    for rh_adv in advisory_data["advisories"]:
+        # RH provides 3 types of advisories RHSA, RHBA, RHEA. Only RHSA's contain severity score.
+        # See https://access.redhat.com/articles/2130961 for more details.
+
+        if "RHSA" in rh_adv:
+            rhsa_data = requests.get(f"https://access.redhat.com/hydra/rest/securitydata/cvrf/{rh_adv}.json").json()  # nopep8
+            value = rhsa_data["cvrfdoc"]["aggregate_severity"]
+            rhsa_aggregate_severity = VulnerabilitySeverity(
+                system=scoring_systems["rhas"],
+                value=value,
             )
-        )
 
-    references.append(Reference(url=advisory_data["resource_url"]))
+            references.append(
+                Reference(
+                    severities=[rhsa_aggregate_severity],
+                    url="https://access.redhat.com/errata/{}".format(rh_adv),
+                    reference_id=rh_adv,
+                )
+            )
+
+        else:
+            references.append(Reference(severities=[], url=url, reference_id=rh_adv))
+
+    redhat_cve_entry = requests.get(advisory_data["resource_url"]).json()
+    redhat_cvss = redhat_cve_entry["cvss3"]["cvss3_base_score"]
+    redhat_cvss3 = VulnerabilitySeverity(
+        system=scoring_systems["rh_cvssv3"],
```

Comment:
Is `rh_cvssv3` really a thing? e.g. are they using rules that are different from the standard `cvssv3`? or rather they just provide a different score using the `cvssv3` rules?
IMHO this is the later, and the `advisory_data["resource_url"])` is enough to qualify where this specific valueof CVSSv3 comes from.

Reply:
>Is rh_cvssv3 really a thing?

yes it is ! https://access.redhat.com/solutions/762393